src/cms/views/pages/page_actions.py

```diff
@@ -109,6 +113,51 @@ def delete_page(request, page_id, region_slug, language_code):
     )
 
 
+@login_required
+def copy_link(request, page_id, region_slug, language_code):
+    """
+    Creates short url of page and copies url to clipboard.
+    """
+    region = Region.get_current_region(request)
+    page = get_object_or_404(region.pages, id=page_id)
+    page_translation = page.get_translation(language_code)
+
+    short_url = (
+        str(request.scheme)
+        + "://"
+        + str(request.get_host())
+        + "/"
+        + str(page_translation.get_short_url())
+    )
+    pyperclip.copy(short_url)
+    messages.success(
+        request,
+        _("URL '{}' was successfuly copied to clipboard.".format(short_url)),
+    )
```

Comment:
```suggestion
    try:
        pyperclip.copy(short_url)
        messages.success(
            request,
            _("URL '{}' was successfuly copied to clipboard.".format(short_url)),
        )
    # pylint: disable=broad-except
    except Exception as e:
        logger.exception(e)
        messages.warning(
            request,
            _("URL '{}' could not be copied to clipboard.".format(short_url)),
        )
```
I couldn't find any resources on the browser support of pyperclip. Even though it worked for me, I think we should at least print the short url even if the copy failed.